src/pytorch_lightning/utilities/rank_zero.py

```diff
@@ -52,52 +44,34 @@ def _get_rank(trainer: Optional["pl.Trainer"] = None) -> Optional[int]:
 
 
 # add the attribute to the function but don't overwrite in case Trainer has already set it
-rank_zero_only.rank = getattr(rank_zero_only, "rank", _get_rank() or 0)
-
-
-def _info(*args: Any, stacklevel: int = 2, **kwargs: Any) -> None:
-    if python_version() >= "3.8.0":
-        kwargs["stacklevel"] = stacklevel
-    log.info(*args, **kwargs)
-
-
-def _debug(*args: Any, stacklevel: int = 2, **kwargs: Any) -> None:
-    if python_version() >= "3.8.0":
-        kwargs["stacklevel"] = stacklevel
-    log.debug(*args, **kwargs)
+new_rank_zero_only.rank = getattr(new_rank_zero_only, "rank", _get_rank() or 0)
 
 
-@rank_zero_only
-def rank_zero_debug(*args: Any, stacklevel: int = 4, **kwargs: Any) -> None:
+@new_rank_zero_only
+def rank_zero_debug(*args: Any, stacklevel: int = 5, **kwargs: Any) -> None:
     """Function used to log debug-level messages only on global rank 0."""
-    _debug(*args, stacklevel=stacklevel, **kwargs)
+    new_rank_zero_deprecation(
+        "`pytorch_lightning.utilities.rank_zero.rank_zero_debug` has been deprecated in v1.8.0 and will be"
+        " removed in v1.10.0. Please use `lightning_lite.utilities.rank_zero.rank_zero_debug` instead."
+    )
+    new_rank_zero_debug(*args, stacklevel=stacklevel, **kwargs)
```

Comment:
We don't want to depreacte these, as the loggers should be different for each library.

This is done in https://github.com/Lightning-AI/lightning/pull/14556 already, just waiting for a new utilities release